robot/impl/local_robot_test.go

```diff
@@ -93,7 +93,7 @@ func TestConfig1(t *testing.T) {
 	c1, err := camera.FromRobot(r, "c1")
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, c1.Name(), test.ShouldResemble, camera.Named("c1"))
-	pic, _, err := camera.ReadImage(context.Background(), c1)
+	pic, err := camera.GoImageFromCamera(context.Background(), rutils.MimeTypeJPEG, nil, c1)
```

Comment:
why are we specifying jpeg here, when previously the `ReadImage` implementation didn't?

Reply:
Had to specify something because otherwise we hit 
```
    /Users/seanyu/Projects/sean-rdk/robot/impl/local_robot_test.go:98: Expected: nil
        Actual:   'could not get image bytes from camera: do not know how to encode ""'
```
Stemming from https://github.com/hexbabe/sean-rdk/blob/65b5bad742bf59c60e249d74f22338c06753465f/components/camera/videosourcewrappers.go#L209